content/cli/overview.md

```diff
@@ -1,23 +1,97 @@
+
+
 
 #### Installation
 
-Install the CLI using **NPM**:
+Install the CLI using **npm**.
+
+> info **Warning** You should use the `-g` option to install `@nestjs/cli` globally, as some of its functions operate outside of the context of a `package.json` file.
 
 ```bash
 $ npm install -g @nestjs/cli
 ```
 
 #### Basic workflow
 
-Once the installation process is finished, you should be able to invoke CLI commands directly from your command line through the `nest` executable. In order to get a list of available functionalities, use the following command:
+Once installed, you can invoke CLI commands directly from your OS command line through the `nest` executable.  See the available `nest` commands by entering the following:
 
 ```bash
 $ nest --help
 ```
 
-#### Contribute
+Get help on an individual command using the following construct. Substitute any command, like `new`, `add`, etc., where you see `generate` in the example below to get detailed help on that command:
+
+```bash
+$ nest generate --help
+```
+
+To create, build and run a new basic Nest project in development mode, go to the folder that should be the parent of your new project, and run the following commands:
+
+```bash
+$ nest new my-nest-project
+$ cd my-nest-project
+$ nest start --watch
+```
+
+In your browser, open [http://localhost:3000](http://localhost:3000) to see the new application running. With the `--watch` flag, the app will automatically recompile and reload when you change any of the source files.
+
+#### Project structure
+
+When you run `nest new`, Nest generates a boilerplate application structure by creating a new folder and populating an initial set of files.  You can continue working in this default structure, adding new components, as described throughout this documentation.  We refer to this as **standard mode**.  Nest also supports the notion of a multi-project **monorepo mode**.
+
+Aside from a very few specific considerations around how the **build** process works (essentially, monorepo mode simplifies build complexities that can sometimes arise from monorepo-style project structures), and built-in [library](/cli/libraries) support, the rest of the Nest features, and this documentation, apply equally to both standard and monorepo mode project structures.  In fact, you can easily switch from standard mode to monorepo mode at any time in the future, so you can safely defer this decision while you're still learning about Nest.
+
+You can use either mode to manage multiple projects. Here's a quick summary of the differences:
+
+| Feature                                              | Standard Mode                                                      | Monorepo Mode                                              |
```

Comment:
This table looks bad on mobile. Not sure what we can do to improve this look & feel though

Reply:
I agree - MD generated tables look bad even on desktop.  I will experiment with HTML tables after we have finalized the content.